products/hashicorp-vault.md

```diff
@@ -16,18 +16,18 @@ versionCommand: vault --version
 releases:
 -   releaseCycle: "1.12"
     eol: false
-    latest: "1.12.2"
-    latestReleaseDate: 2022-11-23
+    latest: "1.12.3"
```

Comment:
Looks like they abandoned the `CHANGELOG.md` method, we may want to update `changelogTemplate` to use GitHub releases instead (https://github.com/hashicorp/vault/releases).